src/Localization/README.md

```diff
@@ -0,0 +1,6 @@
+Localization
+============
+
+These projects provide abstractions for localizing resources in .NET applications.
+
+The ASP.NET Core implementation of localization can be found in https://github.com/aspnet/AspNetCore.
```

Comment:
Maybe we should link directly to https://github.com/aspnet/AspNetCore/tree/master/src/Middleware/Localization since it's not a top-level directory?